coretex/cli/modules/node.py

```diff
@@ -236,6 +236,29 @@ def selectNodeMode(storagePath: str) -> Tuple[int, Optional[int]]:
     return availableNodeModes[selectedMode], None
 
 
+def promptCpu(config: Dict[str, Any], cpuLimit: int) -> int:
+    cpuCount: int = clickPrompt("Enter the number of CPUs the container will use (press enter to use default)", cpuLimit, type = int)
+
+    if cpuCount > cpuLimit:
+        errorEcho(f"ERRROR: CPU limit in Docker Desktop ({cpuLimit}) is lower than the specified value ({cpuCount})")
+        return promptCpu(config, cpuLimit)
+
+    return cpuCount
+
+def promptRam(config: Dict[str, Any], ramLimit: int) -> int:
+    nodeRam: int = clickPrompt("Node RAM memory limit in GB (press enter to use default)", ramLimit, type = int)
```

Comment:
Let's display the min and max values as a part of the prompt. Min value is 6 GB, whilst the max is either ramLimit (if it is set), or the total amount of RAM available for the system.